.github/workflows/main.yml

```diff
@@ -140,20 +140,7 @@ jobs:
   ########################
   # cross compilation
   ########################
-  cross-compile:
-    name: cross compilation
-    runs-on: ubuntu-latest
-    steps:
-      - name: git checkout
-        uses: actions/checkout@v3
-
-      - name: setup go ${{ env.GO_VERSION }}
-        uses: ./.github/actions/setup-go
-        with:
-          go-version: '${{ env.GO_VERSION }}'
-
-      - name: build release for all architectures
-        run: make release
+  # TODO(guggero): revert when the windows itest is working.
```

Comment:
let's do this and let it run and merge it?🤩

Reply:
Done! Changed the base to `0-18-staging-rebased`, as we are dependent on https://github.com/lightningnetwork/lnd/pull/8067 (the `itest_parallel.sh` was only added there).